.github/workflows/build-gradle-project.yml

```diff
@@ -0,0 +1,32 @@
+name: Build this project with gradle
+
+on:
+  push:
+
+jobs:
+  build-gradle-project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: openvenues/libpostal
+          path: libpostal
+      - name: Install libtool
+        run: |
+          cd libpostal
+          ./bootstrap.sh
+          mkdir data_dir
+          ./configure --datadir=`pwd`/data_dir MODEL=senzing
+          sudo make -j4
+          sudo make install
+          sudo ldconfig
+      - uses: actions/checkout@v4
+        with:
+          repository: Qualytics/jpostal
```

Comment:
@albarrentine Shouldn't it be `openvenues/jspostal` instead of `Qualytics/jpostal`, no?